bundle-httpcache/src/main/java/com/adobe/acs/commons/httpcache/store/mem/impl/MemCachePersistenceObject.java

```diff
@@ -88,7 +91,15 @@ public String getContentType() {
      * @return Returned in <code>Map<String, List<String>></code> format.
      */
     public Map<String, List<String>> getHeaders() {
-        return (Map<String, List<String>>) (Map<?, ?>) Multimaps.asMap(headers);
+        Map<String, List<String>> map = new HashMap<>();
```

Comment:
Ran into exceptions; @Sivaramvt  @jasiwalgaurav can you take a look at this and make sure i didnt much anything up?